### PR TITLE
JBEAP-5148: shrinkwrap-resolver - replace obsolete the repository in …

### PR DESCRIPTION
**Pull request by sgilda**
…the custom-settings.xml file

### CODE REVIEW

**Comment by sgilda** [comment]
@MatousJobanek : I tested this and it works. Would you mind reviewing this for me?

**Comment by MatousJobanek** [comment]
Hi,
yeah, this looks great. Thanks @sgilda :rocket:

File: shrinkwrap-resolver/src/test/resources/custom-settings.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 <repository>
                     <id>custom-maven-repo</id>
                     <name>Ibiblio Mirror</name>
```
Comment (MatousJobanek) [inline shrinkwrap-resolver/src/test/resources/custom-settings.xml:30]: @sgilda I haven't noticed that you didn't rewrite the name of the repository. Could you do it to reflect the new custom repository?

Comment (sgilda) [inline shrinkwrap-resolver/src/test/resources/custom-settings.xml:30]: @MatousJobanek : Good catch. Would <name>Maven2</name> suffice?

Comment (MatousJobanek) [inline shrinkwrap-resolver/src/test/resources/custom-settings.xml:30]: Yeah :+1:

**Pull request merged**